client/src/js/modules/types/mx/samples/sample-table-mixin.js

```diff
@@ -292,7 +292,17 @@ export default {
     async createNewSampleGroup(value) {
       this.$emit('update-sample-group-input-disabled', true)
       const sampleGroupModel = new SampleGroup({ NAME: value })
-      await this.$store.dispatch('saveModel', { model: sampleGroupModel })
+      let result;
+      try {
+        result = await this.$store.dispatch('saveModel', { model: sampleGroupModel })
+      } catch (err) {
+        this.$store.commit('notifications/addNotification', { title: 'Error', message: err.message, level: 'error' })
+      }
+      if (result) {
+        const { NAME } = result.toJSON()
+        let message = 'Created sample group ' + NAME
```

Comment:
```suggestion
        const message = 'Created sample group ' + NAME
```

Since this one is a constant (nitpick, shouldn't really have any bearing on the app)